project-templates/import-records/Readme

```diff
@@ -0,0 +1,122 @@
+Requirements
+
+  node >= 4 https://nodejs.org/
+  make      https://www.gnu.org/software/make/
+  jq        https://stedolan.github.io/jq/
+
+Make does not support filenames with special characters unfortunately.  Remove
+special characters in your filenames.
+
+Make is not included on OSX by default, you need to install XCode or use a
+package manager like [Brew](https://brew.sh/) or
+[Nix](https://nixos.org/nix/).
```

Comment:
I actually added a note yesterday in the top level Readme about Make on OSX.  Turns out you just need to run a command to install the Xcode command line utils without installing all of Xcode.  I'll update this Readme also.